Sources/General/Commands/Config.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  Copyright © 2021 Rosberry. All rights reserved.
+//
+
+import Foundation
+import ArgumentParser
+import GeneralKit
+
+final class Config: ParsableCommand {
+
+    private class Print: ParsableCommand {
+
+        // swiftlint:disable:next nesting
+        enum Error: Swift.Error, CustomStringConvertible {
+            case config
+
+            var description: String {
+                switch self {
+                case .config:
+                    return "Cold not read a `.config` file. Try run `general config reset`"
+                }
+            }
+        }
+
+        static let configuration: CommandConfiguration = .init(abstract: "Displays content of .config file")
+
+        required init() {
+            //
+        }
+
+        func run() throws {
+            let config = try self.config()
+            print(config.description)
+        }
+    }
+
+    private class Reset: ParsableCommand {
+        static let configuration: CommandConfiguration = .init(abstract: "Sets .config file to its default value")
+        // swiftlint:disable:next nesting
+        typealias Dependencies = HasFileHelper
+
+        private var dependencies: Dependencies {
+            Services
+        }
+
+        required init() {
+            //
+        }
+
+        func run() throws {
+            try dependencies.fileHelper.removeFile(at: URL(fileURLWithPath: Constants.configPath))
+            print(green(".config file is set to default value"))
```

Comment:
Maybe write out that config file were removed? Default value seems like default config file

Reply:
General does not require .config file. Removing config file means general will use default config values. All related mentions of .config file changed to `general config`.